rskj-core/src/main/java/co/rsk/peg/ReleaseTransactionBuilder.java

```diff
@@ -40,8 +40,8 @@
  */
 public class ReleaseTransactionBuilder {
 
-    public static final int BRIDGE_BTC_TX_VERSION_2 = 2;
-    public static final int BRIDGE_BTC_TX_LEGACY_VERSION = 1;
+    public static final int BTC_TX_VERSION_2 = 2;
+    public static final int BTC_TX_LEGACY_VERSION = 1;
```

Comment:
This one is no longer used, right?

Reply:
It is but only on tests. I let it there for the "before this rskip" assertion checking the version assign is 1.

Reply:
Done. Moved to pegTestUtils.